app/models/grda_warehouse.rb

```diff
@@ -8,4 +8,8 @@ module GrdaWarehouse
   def self.table_name_prefix
     ''
   end
+
+  def self.paper_trail_versions
+    GrdaWarehouse::Version.all
+  end
```

Comment:
I think it is a little nicer to use a module method rather than scattering the class name around.